app/login/utils.ts

```diff
@@ -7,20 +7,27 @@ import { HEADER_HEIGHT } from "@/components/header-constants";
 export const useRootStyles = makeStyles<Theme>((theme) => ({
   root: {
     marginTop: `${HEADER_HEIGHT}px`,
-    backgroundColor: theme.palette.muted.main,
+    minHeight: `calc(100vh - ${HEADER_HEIGHT}px)`,
+    display: "flex",
+    flexDirection: "column",
+  },
+  content: {
+    flexGrow: 1,
+    display: "flex",
+    flexDirection: "column",
   },
   section: {
     display: "flex",
     flexDirection: "column",
     padding: `0 ${theme.spacing(6)}`,
+    backgroundColor: theme.palette.muted.main,
   },
   sectionContent: {
     width: "100%",
     maxWidth: 1400,
     margin: "0 auto",
   },
   noTooltip: {
-    // Disable native tooltip in Safari
     "&::after": {
```

Comment:
@noahonyejese here I'd also keep the comment to know what problem is being solved with the `noTooltip` class.